pyproject.toml

```diff
@@ -88,7 +88,6 @@ minimal = [
     "langchain-community==0.0.27",
     "langchain-openai==0.0.7",
     "langchain_google_vertexai==0.0.5",
-    "langchain-google-genai==0.0.9",
```

Comment:
do we not need this now?

@yadavsahil197 does removal need some extra work now since we are generating the lockfile?

Reply:
No since we only use vertex-ai